src/otx/algo/instance_segmentation/utils/box_ops.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+"""Utilities for bounding box manipulation and GIoU."""
+from __future__ import annotations
+
+import torch
+from torch import Tensor
+from torchvision.ops.boxes import box_area
+
+
+def box_cxcywh_to_xyxy(x: Tensor) -> Tensor:
```

Comment:
Can we use torchvision ops?

Reply:
Maybe, I will take a look the differences.